src/styles.scss

```diff
@@ -1,5 +1,12 @@
 $font-path: '../projects/pastanaga/src/assets/fonts';
 @import "../projects/pastanaga/src/lib/styles/common-reset";
+@import "../projects/pastanaga/src/lib/styles/fonts";
+
+// for custom theming, set CSS variables like this:
```

Comment:
would be good to have this example in the readme as well

Reply:
it is already in the README

Reply:
well, not exactly but good enough :wink: